copier/main.py

```diff
@@ -895,8 +895,12 @@ def _apply_update(self):
             # Try to apply cached diff into final destination
             with local.cwd(subproject_top):
                 apply_cmd = git["apply", "--reject", "--exclude", self.answers_relpath]
+                ignored_files = git["status", "--ignored", "--porcelain"]()
+                # returns "!! file1\n !! file2\n"
+                # extra_exclude will contain: ["file1", file2"]
+                extra_exclude = [filename.split('!! ').pop() for filename in ignored_files.split("\n")][:-1]
```

Comment:
Same thing, more readable:

```suggestion
                extra_exclude = [filename.split('!! ').pop() for filename in ignored_files.splitlines()]
```

Reply:
Good to know :+1: